tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -109,7 +109,9 @@ sub run {
         my $escaped_token = qesap_az_create_sas_token(storage => get_required_var('HANA_ACCOUNT'),
             container => (split("/", get_required_var('HANA_CONTAINER')))[0],
             keyname => get_required_var('HANA_KEYNAME'),
-            lifetime => 30);
+            # lifetime has to be enough to reach the point of the test that
```

Comment:
Any idea what the time units are? If it's seconds, perhaps it should be further increased.

Reply:
It is in minutes: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/qesapdeployment.pm#L1995

It is used in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/qesapdeployment.pm#L2020 in a command like

```
date -u  -d $args{lifetime} minutes '+%Y-%m-%dT%H:%MZ'
```